cli/azd/cmd/down.go

```diff
@@ -137,7 +142,29 @@ func (a *downAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 		return nil, fmt.Errorf("saving environment: %w", err)
 	}
 
-	return nil, nil
+	return &actions.ActionResult{
+		Message: &actions.ResultMessage{
+			Header: "Your Azure resources have been destroyed.",
```

Comment:
@savannahostrowski @Austinauth Does "destroyed" bring in a different meaning than "deleted"? It seems to suggest stronger wording than expected?

Reply:
Agree, we should probably just use delete here.

Reply:
Yep - +1!